src/pages/validator/amplifier/verifier-operations.mdx

```diff
@@ -0,0 +1,46 @@
+# Verifier operations
+
+import { Callout } from "/src/components/callout"
+
+<Callout>
+The Axelar Virtual Machine (AVM) and Amplifier are currently under active development, so these instructions are likely to change. Please check back frequently for updates.
+</Callout>
+
+## Approvals
+
+### Approve a verifier or verifier set
+
+Each verifier in a verifier set must use `ampd` to bond and register chain support themselves. On testnet and mainnet, community governance will call the following to authorize them:
+
+```bash
```

Comment:
This is how you do it if you have permissions, but we need to teach how to do it via governance proposal (AKA how to submit the actual proposal)

Reply:
Refer to https://docs.axelar.dev/learn/cosmwasm-governance